src/config/tomlconfig.rs

```diff
@@ -11,6 +11,8 @@ pub struct TomlConfig {
     pub description: Option<String>,
 
     pub output: Option<TomlOutputConfig>,
+
+    pub use_ace: Option<bool>,
```

Comment:
Yeah, that would be my suggestion. But feel free to disregard it if it does not make sense :smile_cat: